packages/terra-application-utility/src/ApplicationUtility.js

```diff
@@ -0,0 +1,12 @@
+import ApplicationHeaderUtility from './ApplicationHeaderUtility';
+import ApplicationMenuUtility from './ApplicationMenuUtility';
+import UtilityUtils from './_Utils';
```

Comment:
if this is expected to be exported, i think we should remove the underscore from the file name.